Makefile

```diff
@@ -122,9 +122,19 @@ install-glide:
 install: install-glide
 	glide install
 
+.PHONY: install-go-bindata
+install-go-bindata:
+	go get github.com/jteeuwen/go-bindata/...
+	go get github.com/elazarl/go-bindata-assetfs/...
+
 .PHONY: build-examples
-build-examples:
-	go build -o ./examples/hotrod/hotrod-demo ./examples/hotrod/main.go
+build-examples: install-go-bindata
+	cd ./examples/hotrod/services/frontend/ && go-bindata-assetfs -pkg frontend web_assets/... && cd -
```

Comment:
not sure if it works in make, but using parens `(cd x && cmd)` automatically scopes the execution making `cd -` unnecessary